### PR TITLE
Analytical Tracking

### PR DESCRIPTION
**Pull request by jedkirby**
This will enable the very basic analytical tracking of users using the buildpack. It'll only send the app url that's being used, along with a timestamp.

### CODE REVIEW

File: lib/analytics.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+import httplib;
+import json;
+
+class Analytics():
+
+    def __init__(self, domain):
+        self.domain = domain;
+        self.service = 'heroku-buildpack-typekit';
+        self.endpoint = 'jedkirby-develop-pr-51.herokuapp.com';
```
Comment (jedkirby) [inline lib/analytics.py:11]: This will ultimately be the correct domain name; `api.jedkirby.com`. This one is just for testing.

Comment (steadweb) [reply lib/analytics.py:11]: Can you pull this from config?

File: lib/analytics.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        ]);
+
+    ##
+    # Ping the usage of the buildback purely
+    # for statistical information.
+    #
+    # @return bool
+    ##
+    def ping(self):
+        response = self.request('POST', '/api/v1/ping', self.params());
```
Comment (steadweb) [inline lib/analytics.py:62]: Is this endpoint always going to exist?

**Review by steadweb — APPROVED** [review]
Couple of questions added, nothing major.

File: lib/analytics.py (hunk 1 above)
Comment (jedkirby) [reply lib/analytics.py:11]: I could probably do that, but, not sure on the benefits..

File: lib/analytics.py (hunk 2 above)
Comment (jedkirby) [reply lib/analytics.py:62]: Well, _always_ is a big ask, but, hopefully it'll always be there 🙈

**Comment by jedkirby** [comment]
Closing this as it's no longer needed. GitHub/Heroku provide enough statistics already.

**Pull request closed without merging**